src/main/java/seedu/taassist/logic/parser/ClassCommandParser.java

```diff
@@ -3,6 +3,7 @@
 import static seedu.taassist.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
 
 import seedu.taassist.logic.commands.ClassCommand;
+import static seedu.taassist.logic.parser.ParserUtil.parseModuleClass;
```

Comment:
I think the static import should be grouped with line 3 to be consistent with the style guide. You might want to check your project settings regarding the imports too.